bin/serve

```diff
@@ -7,4 +7,10 @@ bin/checkinputs "$@"
 export OX_PROJECT=$1
 echo Serving: $OX_PROJECT
 
-webpack-dev-server --progress --config webpack.config.js
+dashboardSocket=$(grep 'webpackDashboardSocketPort' $OX_PROJECT/configs/base.coffee | sed 's|.*: \(.*\)|\1|')
```

Comment:
yeah it's pretty gross right? apparently it has to run in the command, and can just be in the coffee, so i suppose we could run a node script that loads the env...yah ill look into that. i've been wanting to get [dotenv](https://www.npmjs.com/package/dotenv) in here anyways to make things nicer/more configurable